### PR TITLE
Update translated strings

### PR DESCRIPTION
**Pull request by bwaresiak**
Task/Issue URL: https://app.asana.com/0/414235014887631/1204622631026489/f
Tech Design URL:
CC:

**Description**:

**Steps to test this PR**:

Test localization on relevant screens (Settings, Sync).

**Copy Testing**:

* [ ] Use of correct apostrophes in new copy, ie `’` rather than `'`

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: DuckDuckGo/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -1891,9 +1897,36 @@
...
+/* Caption for a button to delete Sync server data */
+"sync.delete.all.confirm.action" = "Eliminare i dati del server";
+
+/* Message for the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.message" = "Tutti i dispositivi saranno disconnessi e i dati sincronizzati verranno eliminati dal server.";
+
+/* Title of the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.title" = "Eliminare i dati del server?";
+
+/* Caption for a button to remove device from Sync */
```
Comment (SabrinaTardio) [inline DuckDuckGo/it.lproj/Localizable.strings:1912]: This seems to be not actually used

File: DuckDuckGo/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -1891,9 +1897,36 @@
...
+"sync.remove-device.action" = "Rimuovi";
+
 /* No comment provided by engineer. */
 "sync.remove-device.message" = "\"%@\" non potrà più accedere ai tuoi dati sincronizzati.";
 
+/* Title of the dialog to remove device from Sync */
```
Comment (SabrinaTardio) [inline DuckDuckGo/it.lproj/Localizable.strings:1918]: neither is this

File: DuckDuckGo/it.lproj/Localizable.strings [hunk 3]
```diff
@@ -1891,9 +1897,36 @@
...
+/* Message for the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.message" = "Tutti i dispositivi saranno disconnessi e i dati sincronizzati verranno eliminati dal server.";
+
+/* Title of the dialog to confirm deleting Sync server data */
+"sync.delete.all.confirm.title" = "Eliminare i dati del server?";
+
+/* Caption for a button to remove device from Sync */
+"sync.remove-device.action" = "Rimuovi";
+
 /* No comment provided by engineer. */
```
Comment (SabrinaTardio) [inline DuckDuckGo/it.lproj/Localizable.strings:1915]: This is not used

**Review by SabrinaTardio — APPROVED** [review]
LGTM, however there are some strings that are not used anymore but I guess we can remove them later

File: DuckDuckGo/it.lproj/Localizable.strings (hunk 1 above)
Comment (bwaresiak) [reply DuckDuckGo/it.lproj/Localizable.strings:1912]: Isn't this used in `UserText.syncRemoveDeviceConfirmAction` which is referenced in alert?

Comment (SabrinaTardio) [reply DuckDuckGo/it.lproj/Localizable.strings:1912]: yeah but the alert itself seems to not be used anywhere anymore

**Pull request merged**